contribute/architecture/frontend-data-requests.md

```diff
@@ -0,0 +1,51 @@
+# Data requests
+
+[BackendSrv is the core data request service](https://grafana.com/docs/grafana/latest/packages_api/runtime/backendsrv) used by Grafana. This documentation highlights the high level concepts used by `BackendSrv`.
+
+## Request `cancellation`
+> **Note:** Data sources can implement their own cancellation concept. This documentation describes the `request cancellation` included in Grafana.
```

Comment:
```suggestion
This section describes how to cancel requests in Grafana. While data sources can implement their own cancellation concept, we recommend that you use the method we describe here.
```

I don't think this needs to be a note. I changed it so that it works better as a introduction to the section.